lib/grandpa/message_handler.go

```diff
@@ -310,7 +310,11 @@ func (h *MessageHandler) verifyJustification(just *Justification, vote *Vote, ro
 	// verify authority in justification set
 	authFound := false
 	for _, auth := range h.grandpa.Authorities() {
-		if reflect.DeepEqual(auth.Key.AsBytes(), just.AuthorityID) {
+		justKey, err := just.AuthorityID.Encode()
+		if err != nil {
+			return err
+		}
+		if reflect.DeepEqual(auth.Key.Encode(), justKey) {
```

Comment:
why is this changed?

Reply:
DeepEqual was returning false because the types don't match.  I used to work with `auth.Key.AsBytes()`, but AsBytes no longer exists.  So I changed so justKey type matchs auth.Key.Encode ([]uint8)